### ISSUE TITLE
Benji Proxmox Setup

### ISSUE THREAD

**Issue by mrtwister76** [elemental-lf/benji#94]
Hi,

I'm new to Benji, so forgive my ignorance :)

I'm interested in backing up virtual machines from a Proxmox cluster running integrated Ceph storage.
The backup target will be a FreeNAS server via NFS.

Are you running Benji directly on the Proxmox nodes, inside a VM, or something else?

Some people seem to be running Benji from a VM or container, if so, how are you accessing the Ceph storage?
I assume you need to configure the VM/container running Benji to have access to the Ceph network but wanted to make sure I wasn't missing a trick :) 
If so, how have you configured this access in Proxmox?

Thanks

**Comment by elemental-lf** [comment]
I'm not running Proxmox in this context, so I can't give first hand advice. But I think that people are running it directly on the host. See the thread at https://forum.proxmox.com/threads/any-proxmox-ceph-users-interested-in-helping-test-benji.63027/, where someone posted a simple script. It could probably be run in a VM, if you install the necessary Ceph credentials inside the VM and provide the correct network connectivity to the VM, so that it can reach the Ceph components. I know that there are some Proxmox users following the issues here, maybe they can give you some additional hints.

**Comment by aletus** [comment]
Hi, I run a lab proxmox cluster, so I'm running benji in an VM to take advantage of HA and migration.  As mentioned, I had to copy the ceph credentials inside the VM.

Note that I could not use benji to backup the VMs that runs benji itself.  The snapshot for itself just hang and never finish most of the time, even though it works just fine for other VM, containers that are running.  I did not spend too much time going down the rabbit hole of figuring out why.
